docbook/gnc-docbookx.dtd

```diff
@@ -164,7 +164,7 @@ own entity definitions to use across all of our DocBook pages. -->
 <!ENTITY url-yh-hlp "https://help.yahoo.com/">  <!-- Append the desired topic -->
 
 <!-- c) URLs of Authors -->
-<!ENTITY url-selinger "https://www.mathstat.dal.ca/~selinger/">
-<!ENTITY tit-sel-tut "&url-selinger;accounting/tutorial.html">
-<!ENTITY tit-sel-gnc "&url-selinger;accounting/gnucash.html">
+<!ENTITY url-selinger "https://www.mathstat.dal.ca/~selinger/accounting/">
+<!ENTITY tit-sel-tut "<foreignphrase>Tutorial on multiple currency accounting</foreignphrase>">
+<!ENTITY tit-sel-gnc "<foreignphrase>Multiple currency accounting in GnuCash</foreignphrase>">
```

Comment:
Inzwischen denke ich, eleganter wäre es, in den Texten nur &ulink-sel-tut; und &ulink-sel-gnc; zu verwenden und die Herleitung derselben hier zu machen. Aber die Entscheidung überlasse ich dir.

Reply:
Die Kombination von `url` und `title` kriege ich nicht hin:
`<!ENTITY url-sel-tut "<ulink url="&url-selinger;tutorial"><foreignphrase>Tutorial on multiple currency accounting</foreignphrase></ulink>">`
Ich schaffe es nicht die Anführungszeichen zu "maskieren"

Reply:
> Ich schaffe es nicht die Anführungszeichen zu "maskieren"

Falsche Antwort! Welche hast du denn probiert?
Verdoppeln, Escapen mit `\`, double-quote in single-quote und umgekehrt fallen mir spontan ein.

Reply:
I got it! Please have a deeper look at it.po

Reply:
Im deutschen Guide in yelp lauf ich beim Öffnen eines `<part>` auf ein `Unbekannter Fehler`, im Englischen nicht.
Kannst du mal gegenchecken?

Reply:
Als commit message würde ich nur sowas wie `Optimize links to Selinger's docs` nehmen.
Wenn dir der partly revert wichtig ist, in Zeile 3 der message.

Reply:
> Im deutschen Guide in yelp lauf ich beim Öffnen eines `<part>` auf ein `Unbekannter Fehler`, im Englischen nicht. Kannst du mal gegenchecken?

Kann ich nicht reproduzieren. Kannst du den `<part>` genauer eingrenzen?

Reply:
`yelp ghelp:gnucash-guide` öffnet neuerdings aus unerfindlichen Gründen eine falsche Version. Daher hatte ich auch nur Icons statt Bilder.
`yelp /usr/local/share/gnome/help/gnucash-guide/de/gnucash-guide.xml` klappt.